server/game/cards/08.6-SAT/SaltcliffeSailor.js

```diff
@@ -0,0 +1,31 @@
+const DrawCard = require('../../drawcard.js');
+
+class SaltcliffeSailor extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            match: card => card.tokens['gold'] > 0,
+            effect: ability.effects.addKeyword('Stealth')
+        });
+        this.action({
+            title: 'Move 1 gold to character',
+            cost: ability.costs.discardGold(),
+            target: {
```

Comment:
As this ability does not explicitly target (use the word "choose"), it needs a ```type: 'select'``` property so the opponent will not know which character you intend to move the token to before deciding whether to cancel.